### ISSUE TITLE
Analytics in Codemoji

### ISSUE THREAD

**Issue by abusedmedia** [MozillaFoundation/encryption_campaign#242]
We are very interested in monitoring how people are going to use Codemoji.
We've already conducted focused user tests with the prototype but we'd like to extend them with more data.
For that reason we're asking whether is possible to add a gAnalytics tracker within the app, I know you've already used it on Advocacy website therefore, it shouldn't be an issue.
A great possibility would be to create a new tracker for Codemoji only, under your account, this way we can have access only to it, without polluting the website access log, provided that is not a problem for you.

Please let us know if it's ok thus we can start integrating the api calls. 

cc/ @OpenMatt @simonwex @Giorgio-Olivero @michellenebiolo

**Comment by ScottDowne** [comment]
@adamlofting So can we setup metrics in a way that gives both mozilla and todo access to the metrics in a way that fits with our privacy concerns?

cc @valianttry

**Comment by adamlofting** [comment]
@ScottDowne Yes, it can be done. We have to add external people to our register of consultants with access.

**Comment by ScottDowne** [comment]
cc @brettgaylor (in case you're currently not following this issue)

**Comment by adamlofting** [comment]
Just repeating my question from the other duplicate thread... 

What domain/subdomain is this going to be hosted on? I need to know that to get the GA profile setup.

:+1:

**Comment by ScottDowne** [comment]
Teach I believe

**Comment by adamlofting** [comment]
More specifically :smile: ...
- Is it a page on teach? (in which case it uses the teach GA ID and is tracked as part of that)
- Or is it a microsite to track as a standalone thing (e.g. codemoji.teach.mozilla.org)?

**Comment by ScottDowne** [comment]
Chris promises it'll be on teach.mozilla.org/something however it's not
part of teach codebase and is a separate app.

**Comment by valianttry** [comment]
I heard in a meeting this will be in learn.mozilla.org? Maybe @lovegushwa can confirm.... want to make sure this analytics setup happens :)
